Glimmer/Assets/Scripts/EnemyController.cs

```diff
@@ -6,13 +6,12 @@ public class EnemyController : MonoBehaviour
 {
     private void OnTriggerEnter(Collider other)
     {
-        PlayerController playerController = other.gameObject.GetComponent<PlayerController>();
+        DetectionAura playerController = other.gameObject.GetComponent<DetectionAura>();
 
-        if(playerController != null)
+        if (playerController != null)
```

Comment:
I understand this is a prototype, but this code is checking for the win/lose condition, and its in EnemyController...